Directory.Packages.props

```diff
@@ -3,7 +3,7 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="Avalonia" Version="11.0.0" />
+    <PackageVersion Include="Avalonia" Version="11.0.4" />
```

Comment:
This should remain 11.0.0

Reply:
can we test against 11.0.4  ?

Reply:
It's testing against 11.0.4 without this change.

Reply:
![image](https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors/assets/2297442/e897e3b0-b931-4d12-8418-b1fd79aade4e)